src/mongo/delegates/Views.class.php

```diff
@@ -169,6 +169,7 @@ public function getViewForResource($resource,$viewType,$context=null)
         {
             $this->getStat()->increment(MONGO_VIEW_CACHE_MISS.".$viewType");
             $viewSpec = Config::getInstance()->getViewSpecification($this->storeName, $viewType);
+            // TODO: remove this line once discussion closed.
```

Comment:
This check exists in `getViewForResource`, but the equivalent check does not exist in `getViewForResources`.  It seems to me that whatever conditions hold in the formert case must also hold in the latter, so should both implement the check?  I noticed this while attempting a small refactor of the common components of each method.